client/my-sites/upgrades/domain-management/edit/test/mapped-domain.js

```diff
@@ -26,7 +27,8 @@ describe( 'mapped-domain', () => {
 				name: 'neverexpires.com',
 				expirationMoment: null
 			},
-			settingPrimaryDomain: false
+			settingPrimaryDomain: false,
+			translate: identity
```

Comment:
But this one is totally optional, since it looks like we're stubbing a lot of stuff here.